scripts/qa_crowbarsetup.sh

```diff
@@ -4095,6 +4095,12 @@ function oncontroller_testsetup
         fi
     fi
 
+    if [[ $want_monasca_proposal = 1 ]]; then
```

Comment:
I don't like such magic rewritings.. would it be okay to just leave that part out? I think it doesn't serve a lot of purpose.. the smoke tests should still be runnable (they just won't test monasca then).